chore: update docs for helm chart install (#958)
caicloud/cyclone: README.md

## Quick Start

> Make sure you have [Helm](https://helm.sh/) installed ([install guide](https://helm.sh/docs/using_helm/#install-helm)). Then install Cyclone with:

Reviewer: ```suggestion
Make sure [Helm](https://helm.sh/) is installed ([install guide](https://helm.sh/docs/using_helm/#install-helm)), then install Cyclone with:
```
